libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1049,14 +1049,14 @@ bool AP_AHRS_DCM::get_position(struct Location &loc) const
 // return an airspeed estimate if available
 bool AP_AHRS_DCM::airspeed_estimate(float &airspeed_ret) const
 {
-    return airspeed_estimate(_airspeed?_airspeed->get_primary():0, airspeed_ret);
+    return airspeed_estimate(AP::airspeed()?AP::airspeed()->get_primary():0, airspeed_ret);
```

Comment:
should not be checking if non-zero, should explicitly looking for nullptr

Reply:
avoid the double call to singleton:
  auto *airspeed = xxxxx;

Reply:
Fixed.